linkerd/app/src/lib.rs

```diff
@@ -188,24 +188,33 @@ impl Config {
             })
         }?;
 
-        debug!(config = ?oc_collector, "Building client");
-        let oc_collector = {
-            let control_metrics =
-                ControlMetrics::register(registry.sub_registry_with_prefix("opencensus"));
+        debug!(config = ?trace_collector, "Building client");
+        let trace_collector = {
+            let control_metrics = ControlMetrics::register(
+                registry.sub_registry_with_prefix(trace_collector.metrics_prefix()),
+            );
```

Comment:
I think this could be something like
```suggestion
            let control_metrics = ControlMetrics::register(
                trace_collector.metrics_prefix().map(|p| registry.sub_registry_with_prefix(p)).unwrap_or_default(),
            );
```

Reply:
Done.